jsonschema2pojo-ant/src/site/Jsonschema2PojoTask.html

```diff
@@ -408,6 +408,12 @@
         </td>
         <td align="center" valign="top">No (default <code>false</code>)</td>
     </tr>
+    <tr>
+        <td valign="top">customSeparatorCharacters</td>
+        <td valign="top">Separators characters to be used to split JSON Pointers ($ref).
```

Comment:
There's a small typo here (_"Separators characters"_). To make it clear that this is not an array or list, I think this text should mention that it's a string. How about this as a description:

> _A string containing any characters that should act as path delimiters when resolving $ref fragments. By default, `#`, `/` and `.` are used in an attempt to support JSON Pointer and JSON Path._